crates/rhyolite_bevy/Cargo.toml

```diff
@@ -21,7 +21,8 @@ blocking = "1.2"
 crossbeam-channel = "^0.5"
 tracing = "0.1"
 png = "0.17"
-
+winit = "0.28.6"
```

Comment:
Consider a more relaxed version constraint here. "0.28" would be more than enough so we don't have to update this every time there is a minor version update.

Reply:
A version constraint of `0.28.6` means the same as `>=0.28.6, <0.29`, so cargo still allows updating the patch version.